src/Client.php

```diff
@@ -537,6 +537,15 @@ protected function normalizePath(string $path): string
         return ($path === '') ? '' : '/'.$path;
     }
 
+    protected function getEndpointUrl(string $subdomain, string $endpoint): string
+    {
+        if (count($list = explode('::', $endpoint)) === 2) {
+            list($subdomain, $endpoint) = $list;
```

Comment:
use `[]` instead of `list` here

Reply:
Won't this blog up when `endpoint` doesn't contain `::`? Could you add a test for that?

Reply:
> Won't this blog up when `endpoint` doesn't contain `::`? Could you add a test for that?

No it won't. The `$subdomain` property does not have a default so is required. If exploding the `$endpoint` by `::` does not contain exactly 2 items, then it will pass through.

I can add a test if you think it necessary though.

Reply:
Do it like this

```php
[$subdomain, $endpoint] = $list;
```

Reply:
[Symmetric array destructuring](https://www.php.net/manual/en/migration71.new-features.php#migration71.new-features.symmetric-array-destructuring) was introduced in PHP 7.1.

This change will break all tests in PHP 7.0 so I have reverted back to using `list`.

Reply:
Go ahead and use it. Let's up the required version of PHP.

Reply:
Okay, I have reapplied the [] usage, however, I have not made any changes to the travis config or to the PHP version constraint in composer .